cryptoki/src/mechanism/aead.rs

```diff
@@ -89,3 +89,105 @@ impl<'a> GcmParams<'a> {
         self.inner.ulTagBits.into()
     }
 }
+
+/// The GCM generator function for the Initialization Vector
+#[derive(Debug, Clone, Copy)]
+pub enum GeneratorFunction {
+    /// CKG_NO_GENERATE no IV generation is done.
```

Comment:
```suggestion
    /// `CKG_NO_GENERATE` no IV generation is done.
```

This is truly minor but I like to use backticks around stuff like this so that the look okay in the generated docs.